flight/Modules/ManualControl/geofence_control.c

```diff
@@ -91,7 +91,7 @@ int32_t geofence_control_select(bool reset_controller)
 	} else {
 		/* Pick default values that will roughly cause a plane to circle down and */
 		/* a quad to fall straight down */
-		stabilization_desired.Throttle = -1;
+		stabilization_desired.Thrust = -1;
```

Comment:
Looks like this needs to be updated to behave the same as failsafe_control i.e. set thrust to 0 if helicp, -1 otherwise (to prevent heli from shooting downward).